client/my-sites/site-settings/section-guided-transfer.jsx

```diff
@@ -0,0 +1,20 @@
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import { Provider } from 'react-redux';
+
+/**
+ * Internal dependencies
+ */
+import GuidedTransfer from 'my-sites/guided-transfer';
+
+export default class extends Component {
+	render() {
+		return (
+			<Provider store={ this.props.store }>
+				<GuidedTransfer />
+			</Provider>
```

Comment:
Perhaps we could wrap the whole `<SiteSettingsComponent/>` with the redux provider? 
If we add `<Provider>` to the render function of client/my-sites/site-settings/main.jsx, we could remove both this and the exporter's provider.